src/runtype.ts

```diff
@@ -130,8 +131,6 @@ export type Unpack<T> = T extends Runtype<infer U>
 // force Typescript to boil down complex mapped types to a plain interface
 export type Collapse<T> = T extends infer U ? { [K in keyof U]: U[K] } : never
 
-const isPureRuntypeSymbol = Symbol('isPure')
-
```

Comment:
I threw out this isPure symbol because we would need two different types for Meta (one that you pass with `isPure` as boolean and the other that is created by `internalRuntype` with `isPure` as `Symbol`).
And that symbol was not used anyway ([see here](https://github.com/hoeck/simple-runtypes/pull/37/files#diff-8c4d2dea81149b89552e33ec597f4c2dbefe11cb3b3f0d64378eaa7d6c934aafL168)).